src/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Request, Response } from 'express';
+import * as _ from 'lodash';
+import { version } from '../package.json';
+import { getCsvDataStream } from './csv';
+import { TransformsList } from 'adlib';
+import { ServiceError } from './csv/service-error';
+
+export = class OutputCsv2 {
```

Comment:
Did you mean to leave the `2` on the end of this?

Reply:
Nope! thanks for catching it. It's a typo. I have removed  it.